docker/dev/docker-compose.yml

```diff
@@ -0,0 +1,70 @@
+version: "3.6"
+services:
+  nginx:
+    image: nginx:1.17-alpine
+    volumes:
+      - ./../nginx/nginx.conf:/etc/nginx/nginx.conf
+      - ./../..:/var/www
+    ports:
+      - 5000:80
+    networks:
+      - internal
+    depends_on:
+      - php_fpm
+  php_fpm:
+    image: php_fpm
+    build:
+      context: ./../..
+      dockerfile: docker/dev/Dockerfile
+      target: php_fpm
+    user: "${UID}:${GID}"
+    volumes:
+      - ./../..:/var/www
+    environment:
+      MYSQL_HOST: database
+      MYSQL_USER: engelsystem
+      MYSQL_PASSWORD: engelsystem
+      MYSQL_DATABASE: engelsystem
+      PHP_IDE_CONFIG: serverName=engelsystem
+    networks:
+      - internal
+      - database
+    depends_on:
+      - database
+  workspace:
```

Comment:
Same here, `engelsystem-dev-workspace` would be nicer

Reply:
Done